### PR TITLE
chore(STADTPULS-468): truncate cards at 100 chars for description

### PR DESCRIPTION
**Pull request by dnsos**
This was missed in the first PR for STADTPULS-468.

### CODE REVIEW

File: src/components/AccountCard/index.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export interface AccountCardPropType {
   categories?: number[];
 }
 
-const DESCRIPTION_MAX_LENGTH = 50;
```
Comment (ff6347) [inline src/components/AccountCard/index.tsx:16]: Exporting the same thing twice in AccountCard/index.tsx and SensorCard/index.tsx
Would it make sense to have this in one place or are these lengths bound to visual space that is different on each card?

**Review by ff6347 — COMMENTED** [review]
Looks. Good to me. See my question in the line comment

File: src/components/SensorCard/index.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export interface SensorCardPropType extends ParsedSensorType {
 }
 
 const DISPLAYABLE_RECORDS_AMOUNT = 20;
-const DESCRIPTION_MAX_LENGTH = 150;
```
Comment (ff6347) [inline src/components/SensorCard/index.tsx:17]: Exporting the same thing twice in AccountCard/index.tsx and SensorCard/index.tsx
Would it make sense to have this in one place or are these lengths bound to visual space that is different on each card?

File: src/components/AccountCard/index.tsx (hunk 1 above)
Comment (dnsos) [reply src/components/AccountCard/index.tsx:16]: Hmm, even though they have the same limit now, I would still not couple the values here.

The elements and the space on the cards are different, so it is possible that we need to separate them again when we touch the appearance of the cards. IMO, there is enough visual and semantic difference to the cards that justifies keeping the separation of the variables.

File: src/components/SensorCard/index.tsx (hunk 2 above)
Comment (dnsos) [reply src/components/SensorCard/index.tsx:17]: See comment above ⬆️

Comment (ff6347) [reply src/components/SensorCard/index.tsx:17]: 👍🏽

**Review by ff6347 — APPROVED** [review]

**Pull request merged**